test/external-modules/die-with-dignity/src/javaRestTest/java/org/elasticsearch/qa/die_with_dignity/DieWithDignityIT.java

```diff
@@ -95,6 +68,22 @@ public void testDieWithDignity() throws Exception {
         }
     }
 
+    private boolean javaPidExists(String expectedPid) throws IOException {
+        final String jpsPath = PathUtils.get(System.getProperty("runtime.java.home"), "bin/jps").toString();
+        final Process process = new ProcessBuilder().command(jpsPath, "-v").start();
```

Comment:
No need for verbose output if all we're checking for is the PID. You can use `-q` instead for only PID output. That will simplify the parsing below too.